pkg/cluster/config.go

```diff
@@ -296,3 +297,21 @@ func ValidateCIDR(cidr string) error {
 	}
 	return nil
 }
+
+// GetDNS parses a subnet in CIDR notation IP address and prefix length,
+// like "172.30.0.0/16", as defined in RFC 4632 and RFC 4291.
+//
```

Comment:
nit: extra/unneeded line

Reply:
That is intentional, for next comment differentiation.

Reply:
How about this?
```
// GetDNS parses a subnet in CIDR format as defined by RFC 4632 and RFC 4291
// and returns the IP address of the Cluster DNS.
// Example: 172.30.0.0/16 returns 172.30.0.10
```

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/blob/7ae95ad56421563592175cf7b98d9497caf47640/pkg/cluster/config.go#L301